packages/pds/src/sql-repo-storage.ts

```diff
@@ -30,25 +29,17 @@ export class SqlRepoStorage extends RepoStorage {
   // instead of waiting for row locks, we attempt to grab the root skipping over locked rows
   // if we get no result, we wait for 200ms so the previous write has the chance to go through & then try again
   // if the lock is still held, we check (without a row lock) to see the row exists at all
```

Comment:
These may be out of date now.  I think it's worth maintaining a note that it may return `null` even if the head exists.

Reply:
yup good point 👍